agenta-web/src/components/TestSetTable/TestsetTable.tsx

```diff
@@ -155,6 +155,8 @@ const TestsetTable: React.FC<testsetTableProps> = ({mode}) => {
     const [columnDefs, setColumnDefs] = useState<{field: string; [key: string]: any}[]>([])
     const [inputValues, setInputValues] = useStateCallback(columnDefs.map((col) => col.field))
     const [focusedRowData, setFocusedRowData] = useState<GenericObject>()
+    const [writeMode, setWriteMode] = useState(mode)
+    const [duplicateTestsetId, setDuplicateTestsetId] = useState(undefined)
```

Comment:
why are we calling this duplicateTestsetId, why not call it testsetId and set it with the test set id when the mode is edit and undefined when the mode is create. Names should be meaningful. duplicateTestsetId is not